static/usage/split-pane/theming/css-properties/demo.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Spinner</title>
+  <link rel="stylesheet" href="../../../common.css" />
+  <script src="../../../common.js"></script>
+  <script type="module" src="https://cdn.jsdelivr.net/npm/@ionic/core@6/dist/ionic/ionic.esm.js"></script>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@ionic/core@6/css/ionic.bundle.css" />
+  <style>
+    ion-split-pane {
+      --side-width: 350px;
+      --side-max-width: 350px;
+
+      --border: 1px dashed #b3baff;
+    }
+  </style>
+</head>
+  <body>
+    <ion-app>
+      <ion-split-pane when="xs" content-id="main">
+        <ion-menu content-id="main">
+          <ion-header>
+            <ion-toolbar color="tertiary">
+              <ion-title>Menu</ion-title>
+            </ion-toolbar>
+          </ion-header>
+          <ion-content class="ion-padding">
+            Menu Content is 350px wide and has a blue dashed border
```

Comment:
The text here is different from what the example code uses, which looks odd.

Reply:
Good catch! Fixed in https://github.com/ionic-team/ionic-docs/pull/2653/commits/5acf12b8751ce529d78787fa0ed994ee29938d06.